Content.Server/Explosion/EntitySystems/TriggerSystem.cs

```diff
@@ -177,6 +192,9 @@ public void HandleTimerTrigger(EntityUid uid, EntityUid? user, float delay , flo
             active.BeepInterval = beepInterval;
             active.TimeUntilBeep = initialBeepDelay == null ? active.BeepInterval : initialBeepDelay.Value;
 
+            var ev = new ActiveTimerTriggerEvent(uid, user);
+            RaiseLocalEvent(uid, ev, true);
```

Comment:
You're raising this as broadcast but don't use it so please leave args as default.